invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides

```diff
@@ -114,8 +121,10 @@
 
   @media screen and (max-width: @largestTabletScreen) {
     .ui.dropdown {
-      .menu {
-        min-width: @minWidthSmallerScreens;
+      &.action-menu-width {
```

Comment:
this change will break all the other mobile dropdowns `.ui.dropdown.menu` which do not use `action-menu-width` class.

Reply:
This change affects only the dropdowns that use classes `members-dropdown-container` and `access-dropdown-container`. I tested changes on both of them computer and mobile

Reply:
👍 thanks for testing